services/kiali/1.67.0/defaults/cm.yaml

```
@@ -33,6 +33,7 @@ data:
             in_cluster_url: http://jaeger-jaeger-operator-jaeger-query.istio-system.svc.cluster.local:16685
             use_grpc: true
         deployment:
+          priorityClassName: dkp-high-priority
```

based on the kiali CR (https://kiali.io/docs/configuration/kialis.kiali.io/#.spec.deployment.priority_class_name), it looks like this should be:
```suggestion
          priority_class_name: dkp-high-priority
```


we'll also want to set this on the operator itself - https://github.com/kiali/helm-charts/blob/master/kiali-operator/templates/deployment.yaml#LL35C8-L35C8